plantcv/plantcv/visualize/overlay_two_imgs.py

```diff
@@ -48,28 +45,12 @@ def overlay_two_imgs(img1, img2, alpha=0.5, size_img=None):
         img2_ = cv2.cvtColor(img2_, cv2.COLOR_GRAY2BGR)
 
     ## sizing
-    sz_img1 = img1_.shape[0:2]
-    sz_img2 = img2_.shape[0:2]
-
-    # if the desired size is not given, use the larger one
-    if size_img is None:
-        size_img = np.max([sz_img1[0], sz_img2[0]]), np.max([sz_img1[1], sz_img2[1]])
+    # assumption: the sizes of img1 and img2 are the same
+    size_img = img1_.shape[0:2]
 
     # initialize the output image
     out_img = np.zeros(size_img + (3,), dtype='uint8')
 
-    # check if sizes are the same
-    if sz_img1 != size_img:
```

Comment:
if I may, I think we should check that the sizes are the same. if not, raise a fatal error and ask the user to resize one of the images e.g. with tranform subpackage

Reply:
Added, thanks @dschneiderch!